src/seedu/addressbook/commands/ViewCommand.java

```diff
@@ -32,7 +32,7 @@ public CommandResult execute() {
             if (!addressBook.containsPerson(target)) {
                 return new CommandResult(Messages.MESSAGE_PERSON_NOT_IN_ADDRESSBOOK);
             }
-            return new CommandResult(String.format(MESSAGE_VIEW_PERSON_DETAILS, target.getAsTextHidePrivate()));
+            return new CommandResult(String.format(MESSAGE_VIEW_PERSON_DETAILS, target.getAsTextNoRestrictions()));
```

Comment:
Good that you attempted to use the _getPrintableString_ with this new method. However, this changes the behavior of  View command. Please note that going forward you need to update the tests as well when you make any behavior changes.